gulpfile.js

```diff
@@ -192,33 +218,26 @@ gulp.task('styles-grid', function() {
     .pipe($.size({title: 'styles-grid'}));
 });
 
+// Build with Google's Closure Compiler.
+// To run this step, run "bower install" within MDL to fetch the compiler jar,
+// note that Java 1.7+ is required. This is an optional step.
+gulp.task('closure', function() {
+  return gulp.src(SOURCES)
+    .pipe(closureCompiler({
+      compilerPath: 'bower_components/closure-compiler/compiler.jar',
```

Comment:
We don't use bower as a package manager in MDL, so I'm confused by this line. Going from your `closure-compiler` package in `devDependencies`, this path should instead be `'node_modules/closure-compiler/lib/vendor/compiler.jar'`. Or is there something I'm missing?